app/platform/platform.h

```diff
@@ -347,13 +347,19 @@ void* platform_print_deprecation_note( const char *msg, const char *time_frame);
 #define PLATFORM_RCR_REFLASH   0x3
 #define PLATFORM_RCR_FLASHLFS  0x4
 #define PLATFORM_RCR_INITSTR   0x5
+#define PLATFORM_RCR_STARTUP_OPTION   0x6
 #define PLATFORM_RCR_FREE      0xFF
 typedef union {
     uint32_t hdr;
     struct { uint8_t len,id; };
 } platform_rcr_t;
 
+#define STARTUP_OPTION_NO_BANNER        (1 << 0)
+#define STARTUP_OPTION_160MHZ           (1 << 1)
+#define STARTUP_OPTION_DELAY_MOUNT      (1 << 2)
```

Comment:
I might just be missing it, but I don't see the `LROT` entries I expect to see for these to be exported to Lua?

Reply:
They aren't there -- the interface was changed as a result of the discussion, so you now say 

```
node.startup({banner=false})
```

Reply:
I could add constants for node.STARTUP_BANNER = "banner" but I'm not sure that that is really necessary.

Reply:
No, indeed, that's not necessary.  Thank you for reminding me why I don't see the LROT entries. :)